app/components/header.hbs

```diff
@@ -71,6 +71,8 @@
 
           {{#if this.billingStatusDisplay.shouldShowVipBadge}}
             <Header::VipBadge class="mr-3" />
+          {{else if this.billingStatusDisplay.shouldShowMemberBadge}}
```

Comment:
@libmartinito let's show this in the course page header too please, just like with the VIP / free weeks stuff